iModelCore/util/gtest/include/gtest/gtest-printers.h

```diff
@@ -473,11 +473,11 @@ GTEST_API_ void PrintTo(wchar_t wc, ::std::ostream* os);
 
 GTEST_API_ void PrintTo(char32_t c, ::std::ostream* os);
 inline void PrintTo(char16_t c, ::std::ostream* os) {
-  PrintTo(ImplicitCast_<char32_t>(c), os);
+  PrintTo(static_cast<char32_t>(c), os);
```

Comment:
the implicit cast produces compiler warning now because it changes the meaning of the value. static cast is needed to reflect the intent.